.github/actions/aws-openshift-rosa-hcp-dual-region-cleanup/action.yml

```diff
@@ -85,3 +85,10 @@ runs:
               ${{ github.action_path }}/scripts/destroy-clusters.sh "${{ inputs.tf-bucket }}" \
                 ${{ github.action_path }}/../../../aws/openshift/rosa-hcp-dual-region/terraform/ /tmp/cleanup/ \
                 ${{ inputs.max-age-hours-cluster }} ${{ inputs.target }} ${{ inputs.tf-bucket-key-prefix }}
+
+        - name: Upload cleanup logs
```

Comment:
I'd suggest adding
`if: always()`

we do a check as part of the script that may exit 0 or 1 and would be nice to have the logs regardless.